_maps/RandomRuins/IceRuins/icemoon_surface_engioutpost.dmm

```diff
@@ -44,9 +44,8 @@
 	dir = 8
 	},
 /obj/structure/cable,
-/obj/machinery/power/apc/auto_name/directional/east{
-	start_charge = 0
-	},
```

Comment:
you could have made an updatepaths for changes like these, in fact you really should have done than that use find-and-replace or doing it manually (unsure which one). something like

```txt
/obj/machinery/power/apc/@SUBTYPES{start_charge=0} : @OLD{@OLD;start_charge=@SKIP}, /obj/effect/mapping_helpers/apc/no_charge
```

Something like this for all changes introduced in this PR would be greatly appreciated for both maps that are currently in-progress as well as to be courteous to downstreams. You can read more on writing updatepaths here https://github.com/tgstation/tgstation/tree/master/tools/UpdatePaths

Reply:
I did it both ways 
![SMOrc](https://user-images.githubusercontent.com/93882977/231000743-dbf52f24-0ecc-46fd-90e3-c43ac0408eb7.png)